CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 cmake_minimum_required(VERSION 3.1)
 
+set(MAVSDK_VERSION_MAJOR 0)
+set(MAVSDK_VERSION_MINOR 28)
```

Comment:
@caleridas could we get the version from the git tag like we do it here?
https://github.com/mavlink/MAVSDK/blob/53f41d62c0f9e041ba7087dec12da21f29a5503d/src/CMakeLists.txt#L53-L61

Reply:
good point -- one note is that you would still have to keep debian/changelog updated manually, it is defining the package version (which of course should match the library version); it would be neat to have that automated as well, but I have honestly not been able to figure out fully how to do that part

Reply:
Ok, we'll put it in the docs on how to make a release then so we don't miss it:
https://mavsdk.mavlink.io/develop/en/contributing/release.html#mavsdk-part